joomla/joomla-cms
# [4.0] Fix error on click article version history

Pull Request for Issue 
Click on version history of an article.
Error: <code>0 Call to a member function getSha1() on boolean</code>
On linux.

### Summary of Changes

<code>$table</code> name changed to <code>Contenthistory-->ContentHistory</code>


### Testing Instructions

<h4>Test this on Linux.</h4>
And make sure this change also work in Mac and windows.

1.Go to Content ->edit article 
2.Click on a version.

### Expected result
No error comes.


### Actual result
Error comes 

<code>0 Call to a member function getSha1() on boolean</code>

### Documentation Changes Required
